client/layout/guided-tours/tours/editor-basics-tour.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import {
+	overEvery as and,
+} from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Tour,
+	Step,
+	ButtonRow,
+	Next,
+	Quit,
+	Link,
+} from 'layout/guided-tours/config-elements';
+import {
+	isNewUser,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const EditorBasicsTour = makeTour(
+	<Tour
+		name="editorBasicsTour"
+		version="20170503"
+		path="/post/"
+		when={ and( isDesktop, isNewUser ) }
+	>
+		<Step
+			name="init"
+			arrow="top-left"
+			target=".editor-title"
+			placement="below"
+			style={ { animationDelay: '5s', } }
+		>
+			<p>
+				{ translate( 'Welcome to the editor! Add a title here.' ) }
+			</p>
+			<ButtonRow>
+				<Next step="write" />
+				<Quit />
+			</ButtonRow>
+		</Step>
+		<Step
+			name="write"
+			arrow="top-left"
+			target=".mce-toolbar-grp.mce-container"
+			placement="below"
+			style={ { marginTop: '40px' } }
+		>
+			<p>
+				{ translate( 'Write your post in the content area.' ) }
+			</p>
+			<img
+				src="https://en-support.files.wordpress.com/2017/03/editor-content-area_360.gif"
```

Comment:
I recommend going to the relevant Media Library to find these resources and cross-reference either this PR or this file (`client/layout/guided-tours/tours/editor-basics-tour.js`), to minimize risks of having these resources deleted by someone unaware of their use in Calypso.